flink-runtime-web/web-dashboard/package.json

```diff
@@ -4,58 +4,58 @@
   "scripts": {
     "ng": "node node_modules/@angular/cli/bin/ng",
     "start": "ng serve",
-    "build": "ng build --prod --base-href ./",
+    "build": "ng build --configuration production --base-href ./",
     "test": "ng test",
     "lint": "ng lint",
     "proxy": "ng serve --proxy-config proxy.conf.json"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~7.2.0",
-    "@angular/common": "~7.2.0",
-    "@angular/compiler": "~7.2.0",
-    "@angular/core": "~7.2.0",
-    "@angular/forms": "~7.2.0",
-    "@angular/platform-browser": "~7.2.0",
-    "@angular/platform-browser-dynamic": "~7.2.0",
-    "@angular/router": "~7.2.0",
+    "@angular/animations": "~12.1.3",
+    "@angular/common": "~12.1.3",
+    "@angular/compiler": "~12.1.3",
+    "@angular/core": "~12.1.3",
+    "@angular/forms": "~12.1.3",
+    "@angular/platform-browser": "~12.1.3",
+    "@angular/platform-browser-dynamic": "~12.1.3",
+    "@angular/router": "~12.1.3",
     "@antv/g2": "^3.4.10",
     "core-js": "^2.5.4",
     "d3": "^5.9.1",
     "d3-flame-graph": "^4.0.6",
     "d3-tip": "^0.9.1",
     "dagre": "^0.8.4",
-    "monaco-editor": "^0.16.0",
-    "ng-zorro-antd": "^7.2.0",
-    "rxjs": "~6.3.3",
-    "tslib": "^1.9.0",
-    "zone.js": "~0.8.26"
+    "monaco-editor": "^0.20.0",
+    "ng-zorro-antd": "^12.0.1",
+    "rxjs": "^6.5.3",
```

Comment:
This should be >= 6.6 (see https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3)